src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.test.tsx

```diff
@@ -112,7 +117,7 @@ it("shows the premium upgrade cta if the user is not a premium subscriber", () =
   expect(premiumCtas.length).toBe(2);
 });
 
-it("opens and closes the premium upsell dialog", async () => {
+it("opens and closes the premium upsell dialog (premium upsell badge)", async () => {
```

Comment:
I'm not sure I follow this description. Do you mean 

```suggestion
it("opens and closes the premium upsell dialog via the Premium upsell badge", async () => {
```

? (And similarly to the other test below?)

Reply:
Renamed the tests in [7051172](https://github.com/mozilla/blurts-server/pull/3402/commits/7051172e477127a734f530c2dfd0fb02c1797383).